plugin-api/MetaflowPluginAPI.js

```diff
@@ -139,7 +140,7 @@ const Metaflow = {
         event: 'UPDATE_PLUGIN',
         data: {
           slot: PluginInfo.slot,
-          name: PluginInfo.manifest.name,
+          name: PluginInfo.manifest?.name,
```

Comment:
do we need a fallback value here?

Reply:
I don't have a good option for one - it will be `undefined` as is, and that is probably the best.